less/style.less

```diff
@@ -710,6 +714,27 @@ tr[data-field="band"] {
     max-width: none;
 }
 
+/* Event */
+#id_i_boost_stat + .cuteform {
+    .cuteform-elt:not([data-cuteform-val=""]){
+        border-radius: 50%;
+        text-align: center;
+        color: #666666;
+        font-size: 120%;
+        vertical-align: middle;
+        width: 11%;
+    }
+    .cuteform-elt[data-cuteform-val="0"] {
+	background-color: @performanceColor;
+    }
+    .cuteform-elt[data-cuteform-val="1"] {
+	background-color: @techniqueColor;
+    }
+    .cuteform-elt[data-cuteform-val="2"] {
+	background-color: @visualColor;
+    }
```

Comment:
indentation here too. the `background-color`s should have 4 spaces like the block above them.